src/ldp/http/SimpleSparqlUpdateBodyParser.ts

```diff
@@ -34,16 +36,15 @@ export class SimpleSparqlUpdateBodyParser extends BodyParser {
       const sparql = await readableToString(toAlgebraStream);
       const algebra = translate(sparql, { quads: true });
 
+      const metadata = new RepresentationMetadata();
+      metadata.add(CONTENT_TYPE, 'application/sparql-update');
```

Comment:
Shall we actually facilitate this a bit in the constructor, such that we can

```suggestion
      const metadata = new RepresentationMetadata({ [CONTENT_TYPE]: 'application/sparql-update' });
```

or even for the common ones

```suggestion
      const metadata = new RepresentationMetadata({ contentType: 'application/sparql-update' });
```

Reply:
Also on https://github.com/solid/community-server/pull/65/files#diff-f5a8d3f08294700c004106c9a7fcb140R120 and possibly others